src/screens/Login/Login/LoginContainer.js

```diff
@@ -85,7 +85,7 @@ class LoginContainer extends PureComponent {
     } = this.props.navigation.state.params
     const credentials = await Keychain.getInternetCredentials(account.address)
     if (credentials.password === password) {
```

Comment:
To check that password is correct we need to encrypt a wallet with this password. User will be logged in in case of success decryption only.